cfme/tests/test_rest.py

```diff
@@ -517,3 +519,39 @@ def test_edit_requests(self, rest_api, pending_requests, from_detail):
         for request in pending_requests:
             request.reload()
             assert request.options['arbitrary_key_allowed'] == 'test_rest'
+
+    @pytest.mark.uncollectif(lambda: current_version() < '5.7')
+    def test_create_automation_requests_parallel(self, rest_api):
+        """Create automation requests in parallel.
+
+        Metadata:
+            test_flag: rest
+        """
+        output = mp.Queue()
+        entry_point = rest_api._entry_point
+        auth = rest_api._auth
+
+        def _gen_automation_requests(entry_point, auth, output):
+            api = MiqApi(entry_point, auth, verify_ssl=False)
+            requests_data = automation_requests_data(
+                'nonexistent_vm', requests_collection=True, approve=False)
+            api.collections.requests.action.create(*requests_data[:2])
+            result = (api.response.status_code, api.response.json())
+            output.put(result)
+
+        processes = [
+            mp.Process(target=_gen_automation_requests, args=(entry_point, auth, output))
+            for _ in range(4)]
+
+        for proc in processes:
+            proc.start()
+
+        # wait for all processes to finish
+        for proc in processes:
+            proc.join()
```

Comment:
**OPTIONAL**
You may want a JoinableQueue here, it would require calling task_done() in `_gen_automation_requests()` but would allow you to block here with `output.join()`